site/docs/v1/tech/events-webhooks/events.adoc

```diff
@@ -27,6 +27,24 @@ These are the events that FusionAuth generates that can be optionally consumed b
 * <<JWT Refresh>> - when an access token is refreshed using a refresh token
 * <<JWT Refresh Token Revoke>> - when a refresh token (or multiple tokens) are revoked
 
+=== Application Scoped Events
```

Comment:
This is sort of a point of confusion and really is a hold over from this design when it originated in CleanSpeak. We have discussed yanking it all together, or re-working it because any event that is for a "user" is not really application scoped. 

Related issues:
https://github.com/FusionAuth/fusionauth-issues/issues/169
https://github.com/FusionAuth/fusionauth-issues/issues/930

We could mention that this behavior may change as well.